packages/bff/src/model/api/converters/catalogClientApiConverter.ts

```diff
@@ -3,15 +3,26 @@
 import { DescriptorWithOnlyAttributes } from "pagopa-interop-agreement-lifecycle";
 import {
   agreementApi,
+  attributeRegistryApi,
   bffApi,
   catalogApi,
   tenantApi,
-  attributeRegistryApi,
 } from "pagopa-interop-api-clients";
 import { EServiceAttribute, unsafeBrandId } from "pagopa-interop-models";
 import { attributeNotExists } from "../../domain/errors.js";
-import { getTenantEmail, isUpgradable } from "../../modelMappingUtils.js";
+import { getTenantEmail } from "../../modelMappingUtils.js";
 import { catalogApiDescriptorState } from "../apiTypes.js";
+import {
+  getLatestAcriveDescriptor,
+  getNotDraftDescriptor,
+  isAgreementSubscribed,
+  isAgreementUpgradable,
+} from "../../mappers.js";
+import { CompactOrganization } from "../../../../../api-clients/dist/bffApi.js";
```

Comment:
Could you double-check this import statement?

Reply:
👍🏻